Sources/SwiftRefactor/ExpandEditorPlaceholder.swift

```diff
@@ -195,45 +205,72 @@ public struct ExpandEditorPlaceholder: EditRefactoringProvider {
 ///   <#T##String#>
 /// }
 /// ```
-public struct ExpandEditorPlaceholdersToTrailingClosures: SyntaxRefactoringProvider {
+public struct ExpandEditorPlaceholdersToLiteralClosures: SyntaxRefactoringProvider {
```

Comment:
Because this is a change to a public API, we should have a release note entry for it. Could you create a `Release Notes/602.md` file and add the change to it?

Reply:
Done!